conans/test/unittests/client/build/msbuild_test.py

```diff
@@ -145,3 +145,52 @@ def custom_properties_test(self):
                                                                       "MyProp2": "MyValue2"})
         self.assertIn('/p:MyProp1="MyValue1"', command)
         self.assertIn('/p:MyProp2="MyValue2"', command)
+
+    @parameterized.expand([("15", "v141"),
```

Comment:
Why not `"16", "v141"` too? (same for some other tests)

Reply:
added 16